godot-headers/README.md

```diff
@@ -0,0 +1,55 @@
+# godot-headers
+
+This repository contains C headers for
+[**Godot Engine**](https://github.com/godotengine/godot)'s *GDNative Extensions* API.
```

Comment:
I kept the README mostly for the "Updating Headers" instruction. I could cut it down to only that part to remove the references to the headers repo, `3.x` branch, etc. But I figured let's keep it simple for now.